src/openrct2/cmdline_sprite.c

```diff
@@ -14,6 +14,9 @@
  *****************************************************************************/
 #pragma endregion
 
+#pragma warning(disable : 4706) // assignment within conditional expression
```

Comment:
If I remember rightly, its a problem with one of the macros in `jansson.h`. I'll check again tonight.